docs/developer-docs/integrations/bitcoin/index.md

```diff
@@ -3,32 +3,41 @@
+-   **Trading Bitcoin:** Trade bitcoin directly on decentralized exchanges on ICP, without requiring any third-party custody of the assets.
+-   **Decentralization swap:** Using bitcoin to buy tokens in a decentralization swap when an SNS-powered DAO decentralizes a service on ICP.
+-   **Chain Key Bitcoin (ckBTC):** A Bitcoin analogue on the Internet Computer that is backed 1:1 by bitcoin is available on ICP. ckBTC is the easiest way to handle bitcoin on ICP.
 
 ## Examples
-These are only a few examples of how one can use the Bitcoin integration feature. Your imagination is the only limit to the endless range of possibilities being opened up by this feature. This documentation explains how to use this feature in your own dApps. Please also note that the Bitcoin integration API is a low-level API that operates on the level of UTXOs and Bitcoin transactions and is non-trivial to use.
+This documentation explains how to use this feature in your own dapps. The Bitcoin integration API is a low-level API that operates on the level of UTXOs and Bitcoin transactions.
 
-After the Bitcoin mainnet release (general availability release) of the Bitcoin feature, a **chain key Bitcoin (ckBTC)** canister was made available. The ckBTC canister provides on-chain bitcoin on ICP, which looks and feels like wrapped bitcoin but has a much stronger underlying trust model because of its decentralized architecture and using threshold ECDSA instead of bridges. 
+The ckBTC canister provides on-chain bitcoin on ICP, which looks and feels like wrapped bitcoin but has a much stronger underlying trust model because of its decentralized architecture that uses threshold ECDSA instead of bridges. 
 
-The ICP envisions that many people will prefer to use ckBTC instead of our native integration for their projects because of some distinct advantages:
--   **Easier to integrate:** instead of using the low-level Bitcoin integration API, one can simply access the ckBTC ledger like any other ledger on the Internet Computer. The ckBTC ledger adheres to the ICRC-2 token standard for fungible tokens.
--   **Faster and cheaper transfers:** ckBTC can be transferred with the low finality time of the Internet Computer (within seconds) and for a fraction of the cost of a Bitcoin transfer on the Bitcoin network. Using this scheme, only the settlement transfers with the Bitcoin network need to be done on the Bitcoin network, the majority of transfers can be done at high speed and low cost directly on ICP.
+Many developers may prefer to use ckBTC instead of our native integration for their projects because of some distinct advantages:
```

Comment:
```suggestion
Many developers may prefer to use ckBTC instead of IC's native integration for their projects because of some distinct advantages:
```